integrations/gitlab/gitlab_integration/ocean.py

```diff
@@ -81,6 +83,9 @@ async def on_start() -> None:
             integration_config["use_system_hook"],
             token_webhook_mapping,
         )
+
+        asyncio.create_task(event_handler.start_event_processor())
+        asyncio.create_task(system_event_handler.start_event_processor())
```

Comment:
cant you handle the create task inside the start event processor?

Reply:
the create_task here is for it to run in the background and won't block the event loop